docfx_project/ziti/clients/linux.md

```diff
@@ -1,59 +1,147 @@
 ## Linux
 
-The linux tunneler is capable of operating in both transparent and proxy modes. In transparent mode the tunneler will
-interact with the operating system to establish intercept points. These intercept points correspond directly to the
-service definitions in place.
+### The Tunneler CLI
 
-`ziti-tunnel` supports several intercept modes, which are specified with a sub-command.
-Typically you will run `ziti-tunnel` with the _run_ sub-command, which determines
-the preferred intercept mode based on the kernel drivers that are available on the
-host.
+`ziti-edge-tunnel` is the general purpose tunneler CLI and can also run as a systemd daemon. For the best overall experience, please use the preferred tunneler `ziti-edge-tunnel` described here.

```

Comment:
well... I need to figure out if it's "tunneler" (two l's) and "tunneling" (one l) or if "tunneller" is proper. The dictionary one of my editors uses continuously complains about the "one l" variant being misspelled. Word, for example, considers the "one l" variant wrong.  

I'm pretty sure I have tons of doc written with "tunneller" and "tunneling".... 🤔

Reply:
My guess is that "tunneller" is UK, "tunneler" is US. We can assume our style guide will say "Use US spellings", but we can also have exceptions like "Zitit tunnellers are spelled with two Ls". I could also be guessing wrong about which is the US spelling.

Reply:
Spellcheck agrees that singular is "tunneller."